lib/utils/sanitize-html.js

```diff
@@ -195,6 +195,16 @@ export const sanitizeHtml = content => {
     //
     // @see https://developer.mozilla.org/en-US/docs/Web/API/Element/attributes
     filter(node.attributes, ({ name, value }) => {
+      // No javascript Uris allowed
+      if (
+        value
+          .toLowerCase()
+          .trim()
+          .startsWith('javascript')
+      ) {
+        return true;
+      }
```

Comment:
might be best to move this into `isAllowedAttr()` and pass the value in there

```js
function isAllowedAttr( name, value, tagName, tagValue ) { … }
```

this way we can be explicit in our rules. I wouldn't want to eliminate a valid attribute just because it starts with Javascript

Reply:
That sounds better, I also wasn't a fan of returning `true` early there.

Reply:
All set via cdd0d06